src/item.cpp

```diff
@@ -28,7 +28,10 @@
 static const std::string GUN_MODE_VAR_NAME( "item::mode" );
 static const std::string CHARGER_GUN_FLAG_NAME( "CHARGE" );
 static const std::string CHARGER_GUN_AMMO_ID( "charge_shot" );
-
+bool     furred = 0;
+bool     pocketed = 0;
+bool     leather_padded = 0;  //Move these somewhere reasonable
+bool     kevlar_padded = 0;
```

Comment:
item::has_flag("name") wraps looking for a tag in a set in a bool to make things nice and easy too:
```` if(has_flag("pockets")) ````